### PR TITLE
LPD-24881 Add to Editor Screen and the View Screen to check mvcRenderCommandName

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,

May I ask to review my pr please?

including:

https://liferay.atlassian.net/browse/LPD-24508
https://liferay.atlassian.net/browse/LPD-24881

The link for [figma](https://www.figma.com/file/YMcgShbEYvN2XMPPpc154t/LPD-16311-Define-and-manage-Site-default-D%26M-permissions?type=design&node-id=47-7787&mode=design)

Thank you!

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/product/navigation/control/menu/EditFileEntryHeaderProductNavigationControlMenuEntry.java [hunk 1]
```diff
@@ -74,8 +73,17 @@ public boolean isShow(HttpServletRequest httpServletRequest)
 		PortletDisplay portletDisplay = themeDisplay.getPortletDisplay();
 
 		if ((portletDisplay == null) ||
-			!Objects.equals(
-				portletDisplay.getId(), DLPortletKeys.DOCUMENT_LIBRARY_ADMIN)) {
+			(!StringUtil.equals(
+				"/document_library/edit_file_entry",
+				ParamUtil.getString(
+					httpServletRequest,
+					portletDisplay.getNamespace() + "mvcRenderCommandName")) &&
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/product/navigation/control/menu/EditFileEntryHeaderProductNavigationControlMenuEntry.java:80]: A very minor thing: isn't there an extra set of parenthesis here? It is weird that the auto SF didn't catch this.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/product/navigation/control/menu/EditFileEntryHeaderProductNavigationControlMenuEntry.java (hunk 1 above)
Comment (gergelyszalay) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/product/navigation/control/menu/EditFileEntryHeaderProductNavigationControlMenuEntry.java:80]: Hi @adolfopa, thanks for reviewing the pr. This set (!StringUtil actually added by the gw formatSource automatically, or  an other one is the extra? Thx

Comment (adolfopa) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/product/navigation/control/menu/EditFileEntryHeaderProductNavigationControlMenuEntry.java]: Sorry, you're right. Everything is fine; sorry!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**